tools/autograd/derivatives.yaml

```diff
@@ -238,14 +238,14 @@
 
 - name: addmm(Tensor self, Tensor mat1, Tensor mat2, *, Scalar beta=1, Scalar alpha=1) -> Tensor
   self: maybe_multiply(grad, beta.conj())
-  mat1: mm_mat1_backward(grad, mat2, mat1.sizes(), mat1.strides(), mat1.layout(), alpha)
-  mat2: mm_mat2_backward(grad, mat1, mat2.sizes(), mat2.strides(), mat2.layout(), alpha)
+  mat1: mm_mat1_backward(grad, mat2, mat1.sym_sizes(), mat1.sym_strides(), mat1.layout(), alpha)
```

Comment:
Note that these have to be added to replacements: https://github.com/pytorch/pytorch/blob/44a975335e2d08cbbb07df9a1cebe2620f337ed9/tools/autograd/load_derivatives.py#L748 before this is merged otherwise this will be a major memory regression for all users.

Reply:
this is done